### ISSUE TITLE
The Licensing Terms are strange and non-standard

### ISSUE THREAD

**Issue by shlomif** [Kwpolska/python-project-template#4]
Hi! Thanks for all your work, which I wish to use.

IANAL:

According to https://github.com/Kwpolska/python-project-template/blob/master/LICENSE :

```
Python Project Template is licensed under a BSD-like license.  You are free to
relicense your code to another open source license.  If you want to apply a
commercial (a.k.a. proprietary) license, you must contact me first.

HOWEVER, the following files must remain under the BSD license:

* /.pypt/commitlog
* /.pypt/ghrel
* /.pypt/localegen
* /.pypt/PYPT-UPDATE
* /.pypt/README.rst
* /.pypt/LICENSE.PyPT
* /pypt-extras/AUR/hooks/post-release.hook
* /pypt-extras/AUR/AURvm/aurvm_client.py
* /pypt-extras/AUR/AURvm/aurvm_host.py
* /docs/CONTRIBUTING.rst
* /CONTRIBUTING.rst
* /release
```

Now, a https://en.wikipedia.org/wiki/Permissive_software_license such as the 3 clause BSD allows sublicensing under proprietary licences. And once I sublicensed it under a permissive licence, then I can use any other licence.

So your licensing terms are weird, may not be effective, and risk https://en.wikipedia.org/wiki/License_proliferation . Also see:

https://github.com/shlomif/Freenode-programming-channel-FAQ/blob/master/FAQ_with_ToC__generated.md#i-want-to-release-my-code---which-open-source-licence-should-i-use

**Comment by Kwpolska** [comment]
I tried to come up with new wording and licensing terms. The problem with this is the fact that I do not really want to claim copyright over most files in the template (and many of them may not be considered copyrightable), but at the same time I do want to do that for some of the files (the ones listed in the current LICENSE file). This is also the problem faced by Stack Overflow and people who want to use code off that site.

I decided to choose CC0 for the first group and 3-clause BSD for the second. Are these license terms okay in your mind?

> The Python Project Template is provided under two licenses.
> 
> The main license of the Template, all Template-related files, and some of the provided extras, is the 3-clause BSD license. The 3-clause BSD license is a simple open-source license, which requires you to include the text of the license and the copyright line (with Chris Warrick’s name) with all distributions of the software.
> 
> The following files are under the 3-clause BSD license:
> 
> * /.pypt/commitlog
> * /.pypt/ghrel
> * /.pypt/localegen
> * /.pypt/README.rst (and the copy in the repo’s root directory)
> * /pypt-extras/AUR/hooks/post-release.hook
> * /pypt-extras/AUR/AURvm/aurvm_client.py
> * /pypt-extras/AUR/AURvm/aurvm_host.py
> * /docs/CONTRIBUTING.rst
> * /CONTRIBUTING.rst
> * /release
> 
> The remaining files are themselves dual-licensed under the CC0 license. Note that some of the files contain 3-clause BSD license headers. The 3-clause BSD license is the one I recommend for your project, but I waive all copyright claims over these files to the extent permitted by law, as stated in the CC0 license text. This means that you are free to remove the license headers. You are also free to remove those license headers and pick any other license you want, and you are also free to use a commercial license.

**Comment by shlomif** [comment]
@Kwpolska : they should be fine - thanks!

**Comment by Kwpolska** [comment]
Thanks for bringing this to my attention. I just released v2.2.0 with the amended license.

**Comment by shlomif** [comment]
On Sun, 23 Feb 2020 08:39:40 -0800
Chris Warrick <notifications@github.com> wrote:

> Thanks for bringing this to my attention. I just released v2.2.0 with the
> amended license.
> 

Thanks!

-- 

Shlomi Fish       https://www.shlomifish.org/
https://www.shlomifish.org/humour/bits/facts/Emma-Watson/

It always works at the end. Too bad it doesn't work right at the beginning.
    — Shlomi Fish’s relative.

Please reply to list if it's a mailing list post - http://shlom.in/reply .
